tests/test_organizations.py

```diff
@@ -1,9 +1,4 @@
-import json
```

Comment:
Why are these imports deleted?

Reply:
the json import was not being used so I removed it to clean up the file. If this is being used somewhere that you see I'm happy to add it back in.

Reply:
Sounds good 👍